notebooks/src/3-create-stac-item-with-extension.ipynb

```diff
@@ -4,16 +4,26 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Create a STAC Item that Implements the EO Extension Using PySTAC"
+    "# Create a STAC Item that Implements the Extensions Using PySTAC"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the [previous tutorial](/en/tutorials/2-create-stac-catalog-python/index.html), you created a STAC Item and added it to a STAC Catalog. The item you created only implemented the core STAC Item specification. With [extensions](https://github.com/radiantearth/stac-spec/tree/v0.9.0/extensions), we can record more information and add additional functionality to the item. \n",
+    "In the [previous tutorial](/en/tutorials/2-create-stac-catalog-python/index.html), you created a STAC Item and added it to a STAC Catalog. The item you created only implemented the core STAC Item specification. \n",
     "\n",
-    "This tutorial builds off of the knowledge from the previous tutorial. Let's create an item that utilizes a common STAC Extension: the [Electro-Optical (EO) Extension](https://github.com/radiantearth/stac-spec/tree/v0.8.1/extensions/eo). \n",
+    "\n",
+    "## What is a STAC Extension?\n",
+    "\n",
+    "With [extensions](https://github.com/radiantearth/stac-spec/tree/v0.9.0/extensions), we can provide additional metadata or features related to specific types of geospatial data or specialized use cases.\n",
+    "\n",
+    "For instance, beyond just identifying what an image represents, an extension could indicate the estimated cloud cover, sun elevation, resolution, and other distinct attributes. This granularity not only aids users in pinpointing the exact data they need but also assists developers in comprehensively describing multiple datasets. \n",
+    "\n",
+    "This tutorial builds off of the knowledge from the previous tutorial. Let's create an item that utilizes a common STAC Extensions: \n",
+    "- the [Electro-Optical (EO) Extension](https://github.com/radiantearth/stac-spec/tree/v0.8.1/extensions/eo)\n",
+    "- the [view extension](https://github.com/stac-extensions/view)\n",
+    "- the [Projection Extension](https://github.com/stac-extensions/projection)\n",
     "\n",
     "We will continue to use the data from the previous tutorial and will add the item we create into the STAC Catalog you just created. \n",
     "\n",
```

Comment:
Move the following line ("Given that we know the tiff image we are working with is a [WorldView-3](https://worldview3.digitalglobe.com/) image that has earth observation data, we can enable the [EO Extension](https://github.com/radiantearth/stac-spec/tree/v0.8.1/extensions/eo) to add band information.") down to where you start talking about the EO extension. Was in this place before you added the other extensions. Now, it doesn't fit here very well.